packages/quantic/force-app/main/default/lwc/quanticNotifications/quanticNotifications.js

```diff
@@ -68,13 +81,27 @@ export default class QuanticNotifications extends LightningElement {
     );
   }
 
-  get notifications() {
-    return (
-      this.notifyTriggerState?.notifications.map((notification, index) => ({
-        value: notification,
-        id: index,
-      })) ?? []
-    );
+  handleSearchStatusChange() {
+    if (this.searchStatus?.state?.isLoading) {
```

Comment:
You need to handle the error state as well, as in if an error occurs with a search query, the current notifications should be hidden as well.
this is also something you use from the search status state.
It is something to cover with unit tests as well.
<img width="460" alt="Screenshot 2024-12-03 at 11 06 13 AM" src="https://github.com/user-attachments/assets/5548da81-309b-4712-8b68-43566271884a">

Reply:
fixed and tested that as well! 👌 


https://github.com/user-attachments/assets/4d9d8711-963f-40b7-8822-540b7c282c91